### PR TITLE
Use ace in raw couch

### PR DESCRIPTION
**Pull request by orangejenny**
before
screen shot 2018-08-02 at 2 50 48 pm

after
screen shot 2018-08-02 at 2 50 54 pm

@nickpell / @proteusvacuum

### CODE REVIEW

**Review by nickpell — APPROVED** [review]
Nice!

File: corehq/apps/hqadmin/static/hqadmin/js/raw_couch.js [hunk 1]
```diff
@@ -1,22 +1,20 @@
 /* globals CodeMirror, hqDefine */
```
Comment (proteusvacuum) [inline corehq/apps/hqadmin/static/hqadmin/js/raw_couch.js]: think you can remove `CodeMirror` from here

File: corehq/apps/hqadmin/static/hqadmin/js/raw_couch.js [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+                minLines: 3,
+                fontSize: 14,
+                wrap: true,
+                useWorker: false,
             }
-        }
+        );
+        editor.session.setMode('ace/mode/json');
+        editor.setReadOnly(true);
+        editor.session.setValue(JSON.stringify($("#couch-document").data('doc'), null, 4));
```
Comment (proteusvacuum) [inline corehq/apps/hqadmin/static/hqadmin/js/raw_couch.js]: you can reuse `$element` here.

**Review by proteusvacuum — APPROVED** [review]
yay!

**Pull request merged**